.github/workflows/_test-units.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Units
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        description: 'Version of node to use'
+        required: false
+        type: number
+        default: 20
```

Comment:
Do we specifically target v20 or would we rather do `20.x` as in most of the other workflows?